packages/itwinui-css/src/tree/tree.scss

```diff
@@ -10,6 +10,7 @@ $iui-expander-button-width: ($iui-icon-default) + ($iui-expander-inline-padding
 @mixin iui-tree {
   @include iui-reset;
   list-style: none;
+  border-radius: $iui-border-radius;
```

Comment:
the background is set on the tree so it makes sense

Reply:
amm, its not.
It is set only in `html` file

Reply:
ok i see the background is set on the html file, _but_ i feel like we should still set border-radius on the tree. If the user sets a background on the tree, it will help. If they don't, it will have no effect.

| Without | With |
| --- | --- |
| ![](https://user-images.githubusercontent.com/9084735/185677134-16a6d93f-fc65-4726-a793-ce434ea671d8.png) | ![](https://user-images.githubusercontent.com/9084735/185677338-ae8522f9-ab32-4972-924b-e20e09d540f9.png) |